src/components/menu/menu-item/menu-item.style.js

```diff
@@ -0,0 +1,243 @@
+import styled, { css } from 'styled-components';
+import { StyledSubmenu, StyledSubmenuTitle, StyledSubmenuBlock } from '../submenu-block/submenu.style';
+import { baseTheme } from '../../../style/themes';
+
+const StyledMenuItemWrapper = styled.div`
+  display: inline-block;
+  font-size: 13px;
+  font-weight: 700;
+  height: 40px;
+  padding: 0px 16px;
+  position: relative;
+  cursor: pointer;
+
+  .carbon-portrait {
+    top: -1px;
+  }
+
+  &:focus {
+    z-index: 1;
+  }
+
+  ${({ menuType, theme }) => menuType === 'primary' && css`
```

Comment:
you could put that on top of the styled component:
`$({ menuType, theme, divide, hasSubMenu }) => css`
and then use it as just:
`$(menuType === 'primary' && css`